Makefile

```diff
@@ -69,6 +69,7 @@ deps-test:
 
 # (Re)install the tool
 install: spec
+	$(PIP_INSTALL) -U pip wheel
```

Comment:
```suggestion
	$(PIP) install -U pip wheel
```

Reply:
Done.

Reply:
Adding wheel to `requirements.txt` did not work for me because that comes too late.

Reply:
> Adding wheel to `requirements.txt` did not work for me because that comes too late.

You mean `pip install -r requirements.txt` instead of `pip install -U pip wheel` does not work? I don't understand... (But it's probably too insignificant.)